components/ai_chat/resources/untrusted_conversation_frame/components/regenerate_answer_menu/index.tsx

```diff
@@ -68,6 +77,7 @@ export function RegenerateAnswerMenu(props: Props) {
       >
         <div className={styles.anchorButtonContent}>
           <span className={styles.anchorButtonText}>{modelDisplayName}</span>
+          {turnNEARVerified && <NearLabel />}
```

Comment:
As an additional safety maybe we should also conditionally check that `model.isNearModel`